server/controllers/admin/suppliers.js

```diff
@@ -124,5 +124,48 @@ exports.update = function update(req, res, next) {
   })
   .catch(next)
   .done();
-};
+}
+
+// GET /SUPPLIER/SEARCH
+
+function search(req, res, next) { 
+  var sql;
+  var limit = Number(req.query.limit);
+
+  var name = req.query.name;   
+  var condition = '%' + name + '%';
+
+  sql =
+    'SELECT supplier.uuid, supplier.creditor_uuid, supplier.name, supplier.address_1, supplier.address_2, supplier.email, ' +
```

Comment:
Do you need all these properties to be sent back to the component?  You are sending the entire table!  The component only needs to send to account id back.